hikari/impl/entity_factory.py

```diff
@@ -1580,6 +1598,7 @@ def deserialize_partial_message(  # noqa CFQ001 - Function too long
             message_reference=message_reference,
             referenced_message=referenced_message,
             flags=message_models.MessageFlag(payload["flags"]) if "flags" in payload else undefined.UNDEFINED,
+            stickers=stickers,
```

Comment:
Stickers are not in the object

Reply:
as in the json object or the marshalled object?

Reply:
`PartialMessage` in this case

Reply:
nvm, for some reason github didnt show me that change